pre-backup

```diff
@@ -137,39 +137,25 @@ extract_versioned_apis() {
 
 validate_kinds() {
   local -a expected=(
-    buildconfigs
-    builds
     configmaps
-    cronjobs
```

Comment:
Why no `cronjobs` ?

Reply:
This needs refinement. However I would prefer to leave this out of scope for this PR. There will be another one to fix issues when using this script for OpenShift 4.